server/middleware.ts

```diff
@@ -73,7 +73,10 @@ export function isUserOrAdmin(request: express.Request, response: express.Respon
 export function isAdmin(request: express.Request, response: express.Response, next: express.NextFunction) {
 	response.setHeader("Cache-Control", "private");
 	let user = request.user as IUser;
-	if (!request.isAuthenticated()) {
+	if (request.query.adminKey === config.secrets.adminKey) {
```

Comment:
Are you sure you want this admin-level access information in a query string? It may be behind HTTPS but will probably appear in server logs / history logs. A cookie or `Authorization` header might be better.

Reply:
good call